cpp/cmake/templates/UseDOLFIN.cmake.in

```diff
@@ -34,13 +34,11 @@ if (NOT DOLFIN_USE_FILE_INCLUDED)
   find_package(Boost 1.48 QUIET REQUIRED COMPONENTS
```

Comment:
Maybe we should bump the required version to the minimum required for the Python bindings, which I think is 1.55.